pmm-app/tests/DbaaS/pages/dbaasPage.js

```diff
@@ -0,0 +1,151 @@
+const { I } = inject();
+const assert = require('assert');
+
+module.exports = {
+  url: 'graph/d/pmm-dbaas/dbaas?orgId=1',
+  addedAlertMessage: 'Cluster was successfully registered',
+  confirmDeleteText: 'Are you sure that you want to unregister this cluster?',
+  deletedAlertMessage: 'Cluster successfully unregistered',
+  configurationCopiedMessage: 'Copied',
+  monitoringWarningMessage: 'If you want to use monitoring, you need to set your PMM installation public address in',
+  requiredFieldError: 'Required field',
+  valueGreatThanErrorText: (value) => `Value should be greater or equal to ${value}`,
+  tabs: {
+    kubernetesClusterTab: {
+      addKubernetesClusterButton: '$kubernetes-new-cluster-button',
+      addKubernetesClusterButtonInTable: '//div[@data-qa=\'table-no-data\']//span[contains(text(), \'Register new Kubernetes Cluster\')]',
+      actionsLocator: (clusterName) => `//td[contains(text(), "${clusterName}")]//parent::tr//button[@data-qa="dropdown-menu-toggle"]`,
+      closeButton: '$modal-close-button',
+      clusterConfigurationText: '//div[@data-qa=\'pmm-overlay-wrapper\']//pre',
```

Comment:
also, could you please replace single quotes escaping ( `\'` ) to double quotes ( `"` )

Reply:
Done